import-service/services/apiS3.ts

```diff
@@ -28,10 +29,13 @@ class ApiS3 {
 
       const results = [];
       const data = await new Promise((resolve, reject) => {
-        s3Stream.pipe(csv())
-        .on('data', (data: any) => { results.push(data) })
+        s3Stream.pipe(csv({ separator: ';' }))
+        .on('data', (data: any) => {
+          results.push(data);
```

Comment:
It seems that this is a right place for sending messages (instead of pushing it into array and handle it in another place)

Reply:
Yes, actually i think about it. I will fix it.